### PR TITLE
{bio}[foss/2018b] LOHHLA 2018.11.05 w/ R-3.5.1 (REVIEW)

### PR DESCRIPTION
**Pull request by hajgato**
Needs PR #7210

### CODE REVIEW

**Comment by akesandgren** [comment]
Kicking Travis now that #7210 is merged and because EasyBuild testing has changed.

**Comment by akesandgren** [comment]
Travis kicked

File: easybuild/easyconfigs/l/LOHHLA/LOHHLA-2018.11.05-foss-2018b-R-3.5.1.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+easyblock = 'PackedBinary'
+
+name = 'LOHHLA'
+version = '2018.11.05'
+commit_id = '61f4675'
```
Comment (akesandgren) [inline easybuild/easyconfigs/l/LOHHLA/LOHHLA-2018.11.05-foss-2018b-R-3.5.1.eb]: Needs "local_" prefix nowadays.

Comment (hajgato) [reply easybuild/easyconfigs/l/LOHHLA/LOHHLA-2018.11.05-foss-2018b-R-3.5.1.eb]: Done!

**Comment by akesandgren** [comment]
Test report by @akesandgren
**FAILED**
Build succeeded for 4 out of 6 (1 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/62f39a79ded8f1b13c21375f2d37fb65 for a full test report.

**Comment by zao** [comment]
Test report by @zao
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
freja - Linux fedora 30, Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz, Python 2.7.16
See https://gist.github.com/05c8eb8c162ea870089419f6a665eee1 for a full test report.

**Comment by akesandgren** [comment]
Going in, thanks @hajgato!

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
bear-pg0211u03a.bear.cluster - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz, Python 2.7.5
See https://gist.github.com/43838d513b691cd57124eb79e394944f for a full test report.

**Pull request merged**